helmfile/values/fluentd-user.yaml.gotmpl

```diff
@@ -28,6 +28,7 @@ secret:
 env:
   LIVENESS_THRESHOLD_SECONDS: 900
   STUCK_THRESHOLD_SECONDS: 1200
+  OUTPUT_PORT: 443
 
 additionalPlugins:
   # New version due to https://github.com/fluent/fluent-plugin-prometheus/issues/119
```

Comment:
Still relevant?

Reply:
> Still relevant?

Yes, if this is left out the port will default to 9200

Reply:
My comments was referring to the code below were we install a newer version of the prometheus plugin